hibernate-core/src/main/java/org/hibernate/bytecode/internal/bytebuddy/ByteBuddyState.java

```diff
@@ -211,36 +205,11 @@ void clearState() {
 			}
 		}
 
-		if ( System.getSecurityManager() != null ) {
-			// we authorize the proxy class to access the method lookup dispatcher
-			HibernateMethodLookupDispatcher.registerAuthorizedClass( unloadedClass.getTypeDescription().getName() );
-		}
+		classRewriter.registerAuthorizedClass( unloadedClass.getTypeDescription().getName() );
```

Comment:
Seems you'll be calling `,getTypeDescription().getName()` even when the security manager is not enabled, while previously you didn't.
I don't know if it matters (performance?), but I thought I'd warn you.

Reply:
That's a good remark indeed. I updated my PR, could you check?

Reply:
Looks good to me if that's all you changed.